src/domain/landingPage/landingPageSearchSection/LandingPageSearchSection.tsx

```diff
@@ -66,10 +69,16 @@ const Search: React.FC<SearchProps> = ({
     goToSearchPage(search);
   };
 
-  const handleCategoryClick = (category: CategoryOption) => {
+  const handleCategoryClick = (category: CategoryExtendedOption) => {
+    //categories are extended to have hobby types, which has a different search query parameter.
+    const categoryParam = Object.values(COURSE_HOBBY_TYPES).includes(
+      category.value as COURSE_HOBBY_TYPES
+    )
+      ? 'hobbyTypes'
+      : 'categories';
```

Comment:
Maybe we should consider creating some constants for these query param strings.. If we happen to change the naming we easily forget to update them here.

Reply:
I'm thinking something like this:

_eventSearch/types.ts_
```

export const HOBBY_TYPES_PARAM = 'hobbyTypes';

export interface Filters {
  categories: string[];
  [HOBBY_TYPES_PARAM]?: string[];
  dateTypes: string[];
  divisions: string[];
  end: Date | null;
  isFree?: boolean;
  keyword?: string[];
  keywordNot?: string[];
  onlyChildrenEvents?: boolean;
  onlyEveningEvents?: boolean;
  alsoOngoingCourses?: boolean;
  places: string[];
  publisher?: string | null;
  start: Date | null;
  text: string[];
  audienceMinAgeGt?: string;
  audienceMaxAgeLt?: string;
}
```

Reply:
Of course all of them could be constants!

Reply:
Good example in linkedcomponents-ui by @jorilindell 

https://github.com/City-of-Helsinki/linkedcomponents-ui/blob/develop/src/domain/eventSearch/types.ts
https://github.com/City-of-Helsinki/linkedcomponents-ui/blob/develop/src/domain/eventSearch/constants.ts

Reply:
fixed